resources/js/components/sponsors/SponsorLogin.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <div class="container">
+    <div class="d-flex justify-content-center p-4 ">
+        <img 
+            src="/img/ic_robobuzz_white_outline.svg"
+            alt="RoboJackets Logo"
+            height="200"
+            />
+    </div>
+  <div class="card shadow-sm p-4">
+    <h3 class="text-center mb-4">Sponsor Login</h3>
+
+    <form @submit.prevent="handleSubmit">
+      <transition name="slide-fade" mode="out-in">
+        <div v-if="!emailValidated" key="email-step">
+          <label for="email" class="form-label">Email</label>
+          <input 
+            type="email" 
+            v-model="email" 
+            class="form-control" 
+            placeholder="you@company.com" 
+            required 
+            @keydown.enter.prevent="validateEmail">
+          <button type="button" class="btn btn-primary mt-3" @click="validateEmail">Next</button>
+        </div>
+
+        <div v-if="emailValidated" key="password-step">
+          <label for="password" class="form-label">One-Time Password</label>
```

Comment:
We may need some instructions on this screen to check your email for the one-time password that was just sent

it's possible there's some bootstrap or other components already available to implement this flow faster too